acceptance/Rakefile

```diff
@@ -27,9 +31,11 @@ module HarnessOptions
     :add_el_extras => true,
     :preserve_hosts => 'onfail',
     :forge_host => 'forge-aio01-petest.puppetlabs.com',
-    :'master-start-curl-retries' => 30,
+    :'master-start-curl-retries' => 30
```

Comment:
FWIW I'm fond of the trailing comma on the last item in a multi-line construct (because of shorter diffs when extending the construct).

But regardless, I'd prefer to keep this commit's diff just the functional changes, and then if we want to do a style scrub for trailing commas across the code base that could be a separate PR (ideally rubocop-backed so that it would  be enforced).